src/pages/TestBallotDeckScreen.tsx

```diff
@@ -122,10 +122,10 @@ const TestBallotDeckScreen = ({
 
   return (
     <React.Fragment>
-      <Main>
-        <MainChild maxWidth={false}>
-          {ballots.length ? (
-            <React.Fragment>
+      {ballots.length ? (
+        <React.Fragment>
```

Comment:
Somewhat unrelated to this PR, but I just noticed it here. Do you have a preference between `<React.Fragment>…</React.Fragment>` and `<></>`?

Reply:
I do prefer `<React.Fragment>…</React.Fragment>` because it's more clear/obvious to me at a glance. But I do tend to prefer more concise syntax, so I could be convinced to use the shorter syntax.